build/helper/metadata_filters.py

```diff
@@ -301,6 +301,11 @@ def filter_codegen_functions(functions):
     return {k: v for k, v in functions.items() if v['codegen_method'] != 'no'}
 
 
+def filter_library_functions(functions):
+    '''Returns function metadata only for those functions to be included in codegen'''
```

Comment:
A little confusing, because we do code generate for the 'api' only functions. Just not in the library.
Maybe reword: Returns function metadata only for those functions we want to include in the Library class. ?

Reply:
Add clarification text: `Returns function metadata only for those functions to included the library layer (library.py and mock_helper.py)`